src/payment/PaymentPage.test.jsx

```diff
@@ -117,9 +147,11 @@ describe('<PaymentPage />', () => {
     it('should render the basket with an enterprise offer', () => {
       const component = (
         <IntlProvider locale="en">
-          <Provider store={store}>
-            <PaymentPage />
-          </Provider>
+          <AppContext.Provider value={{ authenticatedUser }}>
```

Comment:
Is it the case that all these usages of Provider don't rely on the `config` too?  We're only giving AppContext part of its normal values here... (locale is missing too)

Reply:
Looks like not, but I added locale and config for completeness.